dashboard/src/t5gweb/libtelco5g.py

```diff
@@ -193,7 +193,26 @@ def get_case_number(link, pfilter='cases'):
     return ''
 
 def get_random_member(team):
-    return random.choice(team)
+    '''Randomly pick team member and avoid picking the same person twice in a row'''
+
+    last_choice = redis_get('last_choice')
+    if len(team) > 1: 
+        if len(last_choice) > 1:
```

Comment:
I was testing this and it will error out if there is no last_choice e.g. it is `None`. And if team is large enough, that while loop might be a bit inefficient. How about something like this?
~~~
    if len(team) > 1: 
        if last_choice is not None:
            team = [member for member in team if member != last_choice]
        current_choice = random.choice(team)
    elif len(team) == 1:
        current_choice = team[0]
    else:
        logging.warning("No team variable is available, cannot assign case.")
        current_choice = None
~~~

Reply:
Clever! Updated with these changes

Reply:
:+1: